python/pyspark/ml/stat.py

```diff
@@ -134,6 +134,65 @@ def corr(dataset, column, method="pearson"):
         return _java2py(sc, javaCorrObj.corr(*args))
 
 
+class KolmogorovSmirnovTest(object):
+    """
+    .. note:: Experimental
+
+    Conduct the two-sided Kolmogorov Smirnov (KS) test for data sampled from a continuous
+    distribution.
+
+    By comparing the largest difference between the empirical cumulative
+    distribution of the sample data and the theoretical distribution we can provide a test for the
+    the null hypothesis that the sample data comes from that theoretical distribution.
+
+    :param dataset:
+      a dataset or a dataframe containing the sample of data to test.
```

Comment:
nit: dataset -> Dataset, dataframe -> DataFrame (It's nice to write class names the way they are defined.)